src/main/java/no/nav/veilarbperson/controller/v3/PersonV3Controller.java

```diff
@@ -32,10 +34,16 @@ public class PersonV3Controller {
 
     @PostMapping("/hent-person")
     @Operation(summary = "Henter informasjon om en person fra PDL")
-    public PersonV2Data hentPerson(@RequestBody PersonRequest personRequest) {
+    public PersonV2Data hentPerson(@RequestBody PdlRequest pdlRequest) {
         authService.stoppHvisEksternBruker();
-        authService.sjekkLesetilgang(personRequest.getFnr());
-        return personV2Service.hentFlettetPerson(personRequest.getFnr());
+        authService.sjekkLesetilgang(pdlRequest.getFnr());
+        if (pdlRequest.getBehandlingsnummer() == null) {
+            secureLog.info("Mottok request mot hent-person med behandlingsnummer null");
```

Comment:
Jeg er litt usikker på om vi kanskje ikke skal slette disse før prodsetting, for det kan være greit å vite hvor mange kall vi mottar uten beh.nr

Reply:
can you make some metric for this or just simple log without behandlingsnummer?

Reply:
How do you mean exactly? :) Why without behandlingsnummer?:)